pages/admin/content-manager/[entity_type_slug]/create.js

```diff
@@ -78,72 +102,159 @@ export default function CreateNewEntry({cache}) {
             }
           }
 
+        case SET_SHOW:
+            return {
+              ...state,
+              show: action.payload
+              }
+
+        case SET_SLUG:
+          return {
+            ...state,
+            slug: action.payload
+          }
+
+          case SET_TEMP:
+            return {
+              ...state,
+              temp: action.payload
+            }
+
     }
   };
 
   const [state, dispatch] = useReducer(reducer, initialState);
 
-
-
   useEffect(() => { 
     (async () => {
-      const attributesRaw = await slsFetch(`/api/klaudsolcms/attributes/${entity_type_slug}`);  
-      const attributes = await attributesRaw.json();
+      const valuesRaw = await slsFetch(`/api/${entity_type_slug}`);  
+      const values = await valuesRaw.json();
 
-           
-      dispatch({type: SET_ATTRIBUTES, payload: attributes.map(attribute => {
-        return {
-          name: attribute.attributes_name, 
-          type: attribute.attributes_type,
-        }
-      })});
+      let attributes = [], columns = [], entries = [];
 
-      dispatch({type: SET_FORM_VALUES, payload: attributes.map(attribute => {
-        return {
-          name: attribute.attributes_name, 
-          type: attribute.attributes_type,
-          value: null,
-        }
-      })});
+      entries = values.data[0];
+      columns = Object.keys(values.metadata.attributes);
+      attributes = Object.values(values.metadata);
+
+      dispatch({type: SET_ATTRIBUTES, payload: attributes});
+      dispatch({type: SET_COLUMNS, payload: columns});
+      dispatch({type: SET_ENTRIES, payload: entries});
+      dispatch({type: SET_SLUG, payload: columns[0]});
 
     })();
   }, [entity_type_slug]);
 
-
+  /*
+  const onCreateEntry = useCallback((entries) => {
+    (async () => {
+      try {
+        dispatch({type: LOADING});
+         const response = await slsFetch(`/api/${entity_type_slug}`, {
+          method: 'POST',
+          headers: {
+            'Content-type': 'application/json'
+          },
+          body: JSON.stringify({ entries })
+        })
+        const { message } = await response.json();
+        dispatch({type: SET_SHOW, payload: true})
+      }
+      catch (ex) {  
+        console.error(ex.stack);  
+        alert(ex.stack);
+
+      }finally {
+        dispatch({type: CLEANUP});
+      }
+    })();
+  }, []);*/
+
+  const addSlash = (entry) => {
+    return entry.replaceAll('\'', '\\\'')
+  }
+
+ const createEntry = (entries, attributes, columns) => {
+
+    let temp = [];
+
+    columns.map(col => {
+      entries[`${col}_type`] === 'text' ? temp.push({value_string: entries[col]}) : null;
+      entries[`${col}_type`] === 'textarea' ? temp.push({value_long_string: entries[col]}) : null;
+      entries[`${col}_type`] === 'float' ? temp.push({value_double: entries[col]}) : null;
+    })
+
+    alert(JSON.stringify(temp));
```

Comment:
Kindly remove the alert. It doesn't give a very good user experience to future users of this app.